MAUI/Masked-Entry/Getting-Started.md

```diff
@@ -168,7 +192,7 @@ maskedEntry.PromptChar = '#';
 
 ![.NET MAUI Masked Entry prompt character.](maskedentry_images/net-maui-masked-entry-prompt-char.png)
 
-## Setting the value
+## Step 7: Setting the value
```

Comment:
remove the steps for this topic. Set the topic directly

Reply:
Addressed.